src/frontends/tensorflow/tests/test_models/gen_scripts/generate_saved_model_broadcast_issue.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import sys
+
+import tensorflow as tf
+import numpy as np
+
+# Create the graph and model
+tf.compat.v1.reset_default_graph()
+with tf.compat.v1.Session() as sess:
+    tf_x = tf.constant(np.array([[1, 2]], dtype=np.dtype('i8')))
+    tf_cast = tf.raw_ops.Cast(x=-1, DstT=tf.int64)
+    tf_fill = tf.fill([0, 2], tf_cast, name="FillOperation")
+    tf_fill2 = tf.fill([1, 2], tf_cast, name="FillOperation2")
```

Comment:
I suggest to keep it as is because it is important to do a constant folding of exactly defined graph.

Otherwise I may need more time to lookup for another graph which will have exactly reproduce same behavior as I want to verify.

Moreover, it is good test to check correctness of Const->Result too, as you mentioned before :)